receiver/prometheusreceiver/README.md

```diff
@@ -68,6 +68,11 @@ prometheus --config.file=prom.yaml
 ```shell
 "--feature-gates=receiver.prometheusreceiver.UseCreatedMetric"
 ```
+- `receiver.prometheusreceiver.EnableCreatedTimestampZeroIngestion`: Enables the Prometheus feature flag [created-timestamps-zero-injection](https://prometheus.io/docs/prometheus/latest/feature_flags/#created-timestamps-zero-injection). Currently, this behaviour is disabled by default. To enable it, use the following feature gate option:
```

Comment:
nit: might want to mention why we disable it :)

Reply:
Thanks addressed both of your comments